src/main/java/org/jabref/gui/externalfiles/UnlinkedFilesDialogViewModel.java

```diff
@@ -145,9 +146,11 @@ public void startSearch() {
     }
 
     public void startImport() {
+        Path directory = Paths.get(this.getSearchDirectory().toString()); // Get the base directory
```

Comment:
This is more complicated than it needs to be:
```suggestion
        Path directory = this.getSearchDirectory();